### PR TITLE
Added Map feature, Contributing.md and DockerFile for local running

### PR DESCRIPTION
**Pull request by Ethan-bradley**
The link for database files after the map doesn't work.

### CODE REVIEW

**Review by TangoYankee — APPROVED** [review]

**Comment by Ethan-bradley** [comment]
Does anyone have write access for merging pull requests for this particular repository?

File: templates/WrongFile.html
Comment (TangoYankee) [inline templates/WrongFile.html:6]: As part of the code refractor, we'll need to move all the inline css to a separate file

File: templates/WrongFile.html
Comment (TangoYankee) [inline templates/WrongFile.html:1]: Going forward, we should explore swapping this page for [an error message](https://www.tutorialspoint.com/flask/flask_message_flashing.htm)

File: templates/WrongFileFormat.html
Comment (TangoYankee) [inline templates/WrongFileFormat.html:5]: This may also be reduced to an error message

File: templates/base.html
Comment (TangoYankee) [inline templates/base.html:43]: The series of breaks should be replaced with styling

File: templates/base.html
Comment (TangoYankee) [inline templates/base.html:71]: In order to help with accessibility for people with disabilities, we'll need to add properties to the the link, such as a label/title.

File: templates/complete.html
Comment (TangoYankee) [inline templates/complete.html:19]: This may also be replaced with a flashed message.

File: templates/complete.html
Comment (TangoYankee) [inline templates/complete.html:22]: JavaScript should be moved to a separate folder.

File: templates/complete.html
Comment (TangoYankee) [inline templates/complete.html:24]: Keep access tokens secret. For example, set them in a settings file that is not tracked by git.

File: templates/complete.html
Comment (TangoYankee) [inline templates/complete.html:29]: Avoid magic numbers

File: templates/home.html
Comment (TangoYankee) [inline templates/home.html:1]: rename file to index, to follow standard practice

File: web.py
Comment (TangoYankee) [inline web.py]: Should this be a secret?

File: templates/WrongFile.html
Comment (TangoYankee) [inline templates/WrongFile.html:1]: It appears this file can be replaced with a [flashed message](https://www.tutorialspoint.com/flask/flask_message_flashing.htm)

File: templates/WrongFile.html
Comment (TangoYankee) [inline templates/WrongFile.html:6]: In line styling should be moved to CSS file

File: templates/WrongFileFormat.html
Comment (TangoYankee) [inline templates/WrongFileFormat.html:7]: [Flashed Message](https://www.tutorialspoint.com/flask/flask_message_flashing.htm)

File: templates/authWrong.html
Comment (TangoYankee) [inline templates/authWrong.html:7]: [Flashed Message](https://www.tutorialspoint.com/flask/flask_message_flashing.htm)

File: templates/base.html
Comment (TangoYankee) [inline templates/base.html:44]: Replace breaks with styling

File: templates/base.html
Comment (TangoYankee) [inline templates/base.html:71]: To improve accessibility for people using screen readers, we should add properties such as a label

File: templates/complete.html
Comment (TangoYankee) [inline templates/complete.html:19]: [Flashed Message](https://www.tutorialspoint.com/flask/flask_message_flashing.htm)

File: templates/complete.html
Comment (TangoYankee) [inline templates/complete.html:22]: JavaScript should be moved to a separate file.

File: templates/complete.html
Comment (TangoYankee) [inline templates/complete.html:24]: Access tokens should be kept secret. For example, set it in a settings file that is not tracked by git.

File: templates/complete.html
Comment (TangoYankee) [inline templates/complete.html:29]: Magic Number

File: web.py
Comment (TangoYankee) [inline web.py]: Should GitHub tokens be secret?

File: web.py
Comment (TangoYankee) [inline web.py:19]: Views should be kept in separate views.py file. To the maximum extend possible, they should only execute routing. Any related functions should be called from other files.

File: templates/home.html
Comment (TangoYankee) [inline templates/home.html:1]: Rename to index.html, to follow standard practice.

File: web.py
Comment (TangoYankee) [inline web.py:37]: If there is a function which assumes that it is getting a post call, it should be wrapped in 'elif (request.method == 'POST')

File: web.py
Comment (TangoYankee) [inline web.py:109]: Avoid names like ```finalFile``` that declare a position in a process. 
1) That process might change
2) I have no idea why it's in that position
3) I may not understand the process and telling me a position doesn't help clarify how the process works

Instead, use a name that describes its properties. ie) ```gps_aq_synced_file```

**Review by TangoYankee — APPROVED** [review]
Because this is the old repository, it is simply being used as a reference. Consequently, it's fine if we don't closely follow the workflow. However, when working on the WOEIP repository, I want to make sure we're on the same page. A pull request which affects this many components should be avoided. Instead, they should focus on a single issue that has already been vetted by the team. That way, it is easier to make necessary changes. That said, thank you for your hard work on getting the project working.

File: web.py
Comment (Ethan-bradley) [reply web.py]: Yes, they probably should be, I have been looking into storing it in a pem key. However, there does appear to be an error where every few months the key stops working, though this appears to be because of the way the key is being used. I will investigate this further.

File: templates/WrongFile.html
Comment (Ethan-bradley) [reply templates/WrongFile.html:6]: Yes, sorry for not doing that in the first place. I was planning to move the CSS inline to a separate file sometime this week, however after I do this the style may be more basic than the one currently on display. After that, we probably should decide on a style template for the web app.

File: templates/base.html
Comment (Ethan-bradley) [reply templates/base.html:71]: I am not sure if we will continue to have this link, as the UX team said that the links at the bottom of the page had the potential to be distracting.

**Pull request merged**
